### ISSUE TITLE
Warnings related to visibilities.Unknown_Parameter_357

### ISSUE THREAD

**Issue by rhammen** [BenPru/luxtronik#186]
Running 2023.11.1-Beta, I see the following 2 warnings in the logs:

```
This error originated from a custom integration.

Logger: custom_components.luxtronik2
Source: custom_components/luxtronik2/coordinator.py:344
Integration: Luxtronik
First occurred: 22:55:22 (1 occurrences)
Last logged: 22:55:22

Could not load visibility visibilities.Unknown_Parameter_357
```

```
This error originated from a custom integration.

Logger: Luxtronik.Visibilities
Source: custom_components/luxtronik2/coordinator.py:427
Integration: Luxtronik
First occurred: 22:55:22 (1 occurrences)
Last logged: 22:55:22

Visibility 'Unknown_Parameter_357' not found
```

**Comment by rhammen** [comment]
I downloaded my diagnostics file, and checked the list of visibilties at the end of it.
There is no Unknown_parameter_357 mentioned in the visbilities dict.
Last entry of my visibilties dict is:
      "354  Unknown_Visibility_354                                      ": "None"

Firmware version 3.79.

**Comment by BenPru** [comment]
What is your installed pypi luxtronik version?
Is it 0.3.14?

**Comment by rhammen** [comment]
> What is your installed pypi luxtronik version? Is it 0.3.14?

yes it's 0.3.14

**Comment by BenPru** [comment]
@rhammen Very strange.
Look in the 0.3.14 source:
357 doesn't exist: [L367](https://github.com/Bouni/python-luxtronik/blob/850daa40041a00bbd0d69f3a98845ae29e09f493/luxtronik/visibilities.py#L367)
And unknown Visibilities are create as [f"Unknown_Parameter_{index}"](https://github.com/Bouni/python-luxtronik/blob/850daa40041a00bbd0d69f3a98845ae29e09f493/luxtronik/visibilities.py#L378C46-L378C46)

Why is it different in your environment?

**Comment by rhammen** [comment]
As far as I can tell, the luxtronik library only creates unknown visibilties if they are reported by the heatpump.
And as stated in a previous post: my firmware version is 3.79
My hypothesis is that this visibility.Unknown_Parameter_357 has been added in a firmware version after 3.79. My heatpump therefore doesn't report it, and it is not created by the luxtronik library.

You could add another firmware check on adding this visibilties,
But maybe it is better (more generic) to check if the parameter/visibility  is reported by the heatpump before adding.
That way if more such parameters/visibilities are added in the future, we don't have to this issue every time.

Below my diagnostics file.


[config_entry-luxtronik2-2890470443fa9886c6a2d3acc8261929.json.txt](https://github.com/BenPru/luxtronik/files/13271790/config_entry-luxtronik2-2890470443fa9886c6a2d3acc8261929.json.txt)

**Comment by BenPru** [comment]
I'm planing to go away from Bounis pypi Bouni/python-luxtronik as the releases are to slow.
And using only one Enum with the direkt IDs from Luxtronik Firmware. But it takes time...

**Comment by BenPru** [comment]
Yes it is "just" a warning and a wrong enabled sensor.
I change this behave in the new code.

**Comment by rhammen** [comment]
Issue solved with 2024.9.30.
Thanks!
